app/env.py

```diff
@@ -1,5 +1,7 @@
 import os
 
+JWT_SECRET = os.getenv("JWT_SECRET", "abcd123")
```

Comment:
This looks like a footgun: what if we forget to add the proper secret in our env variables before deploying?
Could this instead be a randomly generated string?